velox/core/QueryConfig.h

```diff
@@ -674,6 +677,14 @@ class QueryConfig {
     return value;
   }
 
+  int32_t sparkPartitionId() const {
+    auto id = get<int32_t>(kSparkPartitionId);
+    VELOX_CHECK(id.has_value(), "Spark partition id is not set.");
```

Comment:
I wonder why don't we use 0 as the default value for this property. It would make it easier to use the library in tests, single-threaded demos, fuzzer, etc.